dstep/translator/Type.d

```diff
@@ -350,9 +352,10 @@ body
         {
             if (children[dimension].kind == CXCursorKind.CXCursor_IntegerLiteral)
             {
+                auto token = tokenInsideSquareBrackets(cursor);
                 auto expansions = context.macroIndex.queryExpansion(children[dimension]);
 
-                if (expansions.length == 1)
+                if (expansions.length == 1 && token.kind == TokenKind.identifier)
```

Comment:
Either I'm completely misunderstanding the Clang AST or there's something wrong with the locations. The cursor that is passed to `context.macroIndex.queryExpansion` is of the kind `CXCursor_IntegerLiteral`. But it's location is, according to Clang, on line 3, column 10 and offset 75. Its spelling is `__FSID_T_TYPE`. How is that an integer literal?